GithubRepository/test/repositoryUtils.test.js

```diff
@@ -0,0 +1,107 @@
+const { filterRepositoriesWithMoreThanFiveStars } = require('../src/repositoryUtils.js');
+
+
+describe('filterRepositoriesWithMoreThanFiveStars', () => {
+    test('should return repositories with more than 5 stars', () => {
```

Comment:
Try not to use the word `should` in the tests also add one more describe at this level IE: 

```
describe('filterRepositoriesWithMoreThanFiveStars', () => {
   describe('when it receives an array of objects', () => {
        test('should return repositories with more than 5 stars', () =>
```